log/sequencer.go

```diff
@@ -72,25 +72,29 @@ func quotaIncreaseFactor() float64 {
 	return QuotaIncreaseFactor
 }
 
+// InitMetrics sets up the metrics for this package. Must be called before calling IntegrateBatch.
+// Can be called more than once, but only the first call has any effect.
```

Comment:
does something still need to be done for the TODO()? 

You're calling your function InitMetrics ), just wanted to check it's not InitJustSomeMetricsButNotAll?

Reply:
I'm not sure if it is, to be honest? I've done nothing to fix that TODO, and it seems like a reasonable TODO to me.

This is likely to require more PRs to fully address...

Reply:
maybe s/the metrics/some metrics/ to avoid confusion.

Reply:
Done